.env.example

```
@@ -71,6 +71,9 @@ USER_MEMORY_CHECKPOINT_EVERY="50"                             # Snapshot frequen
 # AI & tooling provider credentials
 # -----------------------------------------------------------------------------
 OPENROUTER_API_KEY="your_openrouter_api_key_here"             # Required for LLM-powered features (OpenRouter token or proxy key).
+IFS_MODEL="glm-4.5-air"                                        # Default model selection (see config/model.ts for supported keys).
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The IFS_MODEL key should go before the MEMORY_AGENTIC_V2_ENABLED key

(UnorderedKey)